systems/plants/@RigidBodyManipulator/RigidBodyManipulator.m

```diff
@@ -917,10 +917,10 @@
     
     function body_id = findLinkId(model,linkname,robot,error_level)
       % @param robot can be the robot number or the name of a robot
-      % robot=0 means look at all robots
+      % robot<0 means look at all robots
       % @param error_level >0 for throw error, 0 for throw warning, <0 for do nothing. @default throw error
       % @ingroup Kinematic Tree
-      if nargin<3 || isempty(robot), robot=0; end
+      if nargin<3 || isempty(robot), robot=-1; end
```

Comment:
PLEASE CHECK THIS CAREFULLY: this is an interface change.  Currently the world is set to robotnum=0, and with the previous interface, it was impossible to search only in the world frame.  (in practice this was breaking the parsing for some of the older urdfs, but I believe it was an error in our previous interface design).  Also changed in findJointId